app/javascript/packages/pf-icon/src/data/paths.json

```diff
@@ -502,5 +502,29 @@
   "image-inset-40": [
     512,
     "M511.9 32v12a32.09 32.09 0 0 1-32 32h-448a32.09 32.09 0 0 1-32-32V32a32.09 32.09 0 0 1 32-32h448a32.09 32.09 0 0 1 32 32zM326.3 384a32.09 32.09 0 0 0 32-32V160a32.09 32.09 0 0 0-32-32H185.5a32.09 32.09 0 0 0-32 32v192a32.09 32.09 0 0 0 32 32zm153.6 52h-448a32.09 32.09 0 0 0-32 32v12a32.09 32.09 0 0 0 32 32h448a32.09 32.09 0 0 0 32-32v-12a32.09 32.09 0 0 0-32-32z"
+  ],
```

Comment:
`pf-icon` is being updated here, but the checklist item for updating packages has been crossed out.